src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -93,31 +154,64 @@ public RoadAccident getAccidentByIndex(String index){
      * @return
      */
     public Collection<RoadAccident> getAccidentsByLocation(float minLongitude, float maxLongitude, float minLatitude, float maxLatitude){
-        return null;
+        return roadAccidentList.stream()
+        		.filter(r -> r.getLongitude() > minLongitude).filter(r -> r.getLongitude() < maxLongitude)
```

Comment:
Better to use one filter with several conditions - each filter() - do cycle, so in case of one filter will be only one cycle.